projects/canopy/src/lib/pipes/kebab-case/kebab-case.pipe.ts

```diff
@@ -7,6 +7,13 @@ export class LgKebabCasePipe implements PipeTransform {
   transform(str: string): any {
     return !str
       ? str
-      : str.toLowerCase().replace(/\s/g, '-');
+      : str
+        .replace(/[_\-|]{1,}/g, ' ')
+        .replace(/([a-z])([A-Z])/g, function(match, group1, group2) {
```

Comment:
Could you write it as an arrow function please? `.replace(/([a-z])([A-Z])/g, (match, group1, group2) => {`

Reply:
yep, done